cmd/juju/application/config.go

```diff
@@ -350,6 +355,7 @@ func (c *configCommand) getConfig(client applicationAPI, ctx *cmd.Context) error
 		return err
 	}
 	if len(c.keys) == 1 {
+		ctx.Infof("format %v is ignored", c.out.Name())
```

Comment:
I have a strong suspicion that this will confuse people as it will show output any time they ask for something.

Do you think we really need this? Perhaps as debug output?